src/module.ts

```diff
@@ -44,6 +44,13 @@ declare interface SessionOptions {
     * @docs https://github.com/unjs/unstorage
     */
    storageOptions: CreateStorageOptions,
+   /**
+   * Set the cookie domain
```

Comment:
Can you expand this a bit, eg, what does it mean if the domain for a cookie is set vs it being null?

Linking to a relevant part of MDN docs would also be useful here (:

Reply:
Hi, @BracketJohn. Thanks for response. Added a fix, my mistake)

Reply:
No worries, thanks for iterating here with me (: